package.json

```diff
@@ -20,13 +20,17 @@
   "homepage": "https://github.com/EvidentlyCube/metroidvania-compendium#readme",
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged && npm run test"
+      "pre-commit": "lint-staged"
     }
   },
   "lint-staged": {
     "*.ts": [
       "eslint --fix",
       "git add"
+    ],
```

Comment:
I checked and `lint-staged` [uses  `micromatch`](https://github.com/okonet/lint-staged#filtering-files) library for filtering files.

`micromatch` [supports `?`](https://github.com/micromatch/micromatch#why-use-micromatch), so you can have only one configuration key here, `"*.tsx?"` and it should work. 

Also what about that `git add`? If you remove it does it stop the warning that you talked about?

Reply:
After checking out what `git add` does here, according to [this guide](https://www.robertcooper.me/using-eslint-and-prettier-in-a-typescript-project), it looks like it's better to leave it, and simply ignore the warning.

And using `*.tsx?` returned no files that match, but `*.{ts,tsx}` worked

Reply:
`*.{ts,tsx}` is even better, much more clearer to a quick glance